pai-management/k8sPaiLibrary/maintaintool/docker-ce-install.sh

```diff
@@ -17,6 +17,21 @@
 # DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+scriptPath=$1
+
+# Install python for json join tool.
+if command -v python >/dev/null 2>&1; then
+    echo python has been installed. And skip python install.
+else
+    apt-get update
+    apt-get -y install python
+    if command -v python >/dev/null 2>&1; then
+        echo Successfully install python
+    else
+        echo Failed install python
+        exit 1
```

Comment:
>1 [](start = 13, length = 1)

please sync with Bin about this PR. I remember he asked to refine exit code.

Reply:
Sync with Hao Yuan, we only need to care about zore or nonzero. @hao1939

Reply:
For now, a success/failure signal would be a great improvement.